### PR TITLE
add applications component to frontend

### PR DESCRIPTION
**Pull request by nassimerrahoui**
display list of microservices and their basic informations

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
@nassimerrahoui : nice job. But you need to write unit tests, it's mandatory :)

File: src/main/webapp/app/applications/applications.service.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  /** get all gateway routes of applications instances */
+  public findAllGatewayRoute(): AxiosPromise<any> {
+    return axios.get('management/gateway/routes');
+  }
+
+  /** get proxy request of an application instance */
+  public findProxyRequest(gatwayRoute: string): AxiosPromise<any> {
+    return axios.get('gateway/' + gatwayRoute + '/management/info');
```
Comment (fdelbrayelle) [inline src/main/webapp/app/applications/applications.service.ts:19]: It's a detail but there is a type in the word `gateway` here :)

**Comment by pascalgrimaud** [comment]
Some feedbacks:
- it works well, good job @nassimerrahoui 
- missing: applications.service.spec.ts
- `showApplication` is not covered by tests

**Comment by nassimerrahoui** [comment]
> Some feedbacks:
> 
> * it works well, good job @nassimerrahoui
> * missing: applications.service.spec.ts
> * `showApplication` is not covered by tests

It's done :)

**Pull request merged**